### ISSUE TITLE
RAW doesn't work

### ISSUE THREAD

**Issue by uiliannunes** [alex-agency/XYC#29]
I can't open the raw file after copy to pc, I tried open it on Photoshop & Lightroom.. and I failed.. Can you help me?

Seems that the raw file is corrupted. Or something like that.

**Comment by kikislater** [comment]
If you set timelapse from yi app, it produces unreadable compressed RAW.
Tried to desactive timelapse from Yi app and enable Time Lapse from script

**Comment by uiliannunes** [comment]
how can I do this? In the Yi app, there isn't a timelapse option to deactivate.. :(

**Comment by kikislater** [comment]
Yes it exists. You could shot one by one or countdown or interval (this one is timelapse)

**Comment by uiliannunes** [comment]
Yes.. but the problem is.. If I set the option to generate RAW files in the script and take a normal shot... (not timelapse) the raw file is generated, but didn't open.. I searched on the internet many tutorials to generate the raw file.. but I can't make it work yet..

That's sad! :(

Can you help me?
